src/game_of_life/core.clj

```diff
@@ -1,26 +1,17 @@
 (ns game-of-life.core
-    (:require [game-of-life.game :refer :all]))
+    (:require [game-of-life.game :refer :all]
+              [game-of-life.renderer :as r]
+              [clj-launchpad :as launchpad]))
 
-(def initial-grid
-  (random-grid 80 60 0.4))
-
-(defn clear
-  []
-  (print (str (char 27) "[2J"))) ; clear screen
+(def lpad (launchpad/open))
 
-(defn render
-  [grid cell empty]
-  (print (str (char 27) "[;H")) ; move cursor to the top left corner of the screen
-  (println (to-string grid cell empty)))
+(def initial-grid
+  (random-grid 9 8 0.35))
 
 (defn -main
   "Run some thing!"
   []
-  (let [emoji "◻️"]
```

Comment:
Can we keep both drawing methods and have an easy way to switch between them in the `-main`?